Microsoft.Toolkit.Uwp.UI.Controls/TokenizingTextBox/TokenizingTextBox.xaml

```diff
@@ -135,6 +135,12 @@
 
                 <VisualStateGroup x:Name="FocusStates">
                     <VisualState x:Name="Focused">
+                        <Storyboard>
+                            <ObjectAnimationUsingKeyFrames Storyboard.TargetName="FocusVisual" Storyboard.TargetProperty="BorderThickness">
+                                <DiscreteObjectKeyFrame KeyTime="0" Value="{ThemeResource TextControlBorderThemeThicknessFocused}" />
+                            </ObjectAnimationUsingKeyFrames>
+                        </Storyboard>
```

Comment:
Why not use a single `Setter` instead?
It's more compact and should also be slightly more efficient 😊

```suggestion
                        <VisualState.Setters>
                            <Setter Target="FocusVisual.BorderThickness" Value="{ThemeResource TextControlBorderThemeThicknessFocused}" />
                        </VisualState.Setters>
```

Reply:
@Sergio0694 can you update your suggestion to merge in with the existing setter below here: https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3314/files/0b9e8e51dab94584c448dd1916bc09704cde51d0#diff-8e33dd8c3ab5ac2c6fcdca737b694f10R144?

Then I can accept the contribution so you get credit.

But FYI, I just copied this out of the generic.xaml.